script.js

```diff
@@ -1 +1,252 @@
-console.log('yay!');
+// ======Global variables=====================
+let outputBox = null;
+let firstCardElement = null;
+let canClick = true;
+const firstCardColRowIndex = [];
+const gameDurationis3Min = 1000 * 60 * 3;
+const START_GAME = null;
```

Comment:
If `START_GAME` and `TIMES_UP` are both game modes, consider prefixing them with `GAME_MODE`, like `GAME_MODE_START_GAME` so that it's clearer that they're game modes. 

Constants like `GAME_MODE_START_GAME` need strings, e.g. `const GAME_MODE_START_GAME = 'GAME_MODE_START_GAME'`. Otherwise, if both start game and time's up modes have the same value, e.g. `null`, the program won't be able to tell which game mode we're in!